jsonclient/client.go

```diff
@@ -127,9 +127,6 @@ func New(uri string, hc *http.Client, opts Options) (*JSONClient, error) {
 // returned http.Response can be non-nil even when an error is returned,
 // in particular when the HTTP status is not OK or when the JSON parsing fails.
 func (c *JSONClient) GetAndParse(ctx context.Context, path string, params map[string]string, rsp interface{}) (*http.Response, []byte, error) {
-	if ctx == nil {
```

Comment:
This is fine, but not sure the commit belongs in this PR?

Reply:
dropped